src/MendelIHT_utilities.jl

```diff
@@ -1,25 +1,3 @@
-"""
```

Comment:
Is there any reason you moved this to `common.jl`? Not that I need it to be here, but I was trying to keep the Kevin's code as unchanged as possible.